docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-authoring-ha-proc.adoc

```diff
@@ -0,0 +1,42 @@
+* Replace `<template-path>` with the path to the downloaded template file.
+* Use as many `ATTRIBUTE=value` pairs as needed to set the required attributes. You can view the template file to see descriptions for the attributes.
++
+. Set the following mandatory attributes:
+** *Application Name* (`APPLICATION_NAME`): the name of the OpenShift application. It is used in the default URLs for {CENTRAL} and {KIE_SERVER}. OpenShift also uses the application name to create a separate set of deployment configurations, services, routes, labels, and artifacts. You can deploy several applications using the same template into the same project, as long as you use different application names. 
+** *Business Central Server Certificate Name* (`BUSINESS_CENTRAL_HTTPS_NAME`): the name of the certificate in the keystore that you created in <<secrets-central-create-proc>>.
+** *Business Central Server Keystore Password* (`BUSINESS_CENTRAL_HTTPS_PASSWORD`): the password for the keystore that you created in <<secrets-central-create-proc>>.
+** *KIE Server Certificate Name* (`KIE_SERVER_HTTPS_NAME`): the name of the certificate in the keystore that you created in <<secrets-kie-create-proc>>.
+** *KIE Server Keystore Password* (`KIE_SERVER_HTTPS_PASSWORD`): the password for the keystore that you created in <<secrets-kie-create-proc>>.
+** *KIE Server ID* (`KIE_SERVER_ID`): the name of the server template on the {CENTRAL} that the {KIE_SERVER} is to join. Set this parameter to any value. If you do not set it and then the {KIE_SERVER} pod is restarted or scaled, each new instance of the server might join a new template. 
+** *ImageStream Namespace* (`IMAGE_STREAM_NAMESPACE`): the namespace where the image streams are available. If the image streams were already available in your OpenShift environment (see <<imagestreams-file-install-proc>>), the namespace is `openshift`. If you have installed the image streams file, the namespace is the name of the OpenShift project.
++
+You can also set other parameters as necessary. Notably, you can set the following user names and passwords:
+** *KIE Admin User* (`KIE_ADMIN_USER`) and *KIE Admin Password* (`KIE_ADMIN_PWD`): the user name and password for the administrative user in {CENTRAL}.
+** *KIE Server User* (`KIE_SERVER_USER`) and *KIE Server Password* (`KIE_SERVER_PWD`): the user name and password that a client application must use to connect to the {KIE_SERVER}.
++
+If you want to place the built KJAR files into an external Maven repository, set the following parameters:
+** *Maven repository URL* (`MAVEN_REPO_URL`): the URL for the Maven repository. 
+** *Maven repository username* (`MAVEN_REPO_USERNAME`): the username for the Maven repository. 
+** *Maven repository password* (`MAVEN_REPO_PASSWORD`): the username for the Maven repository. 
++
+. Complete the creation of the environment. Depending on the method that you are using:
+.. In the OpenShift Web UI, click *Create*.
+.. Complete and run the command line.
```

Comment:
I would probably go with this:
```
. Complete the creation of the environment depending on the method that you are using:
+
 * In the OpenShift Web UI, click *Create*.
 * In the command line, ... <whatever they need to do>.
```

Reply:
This references the two methods at the beginning of the procedure so uses the same formatting as is used there